packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -1907,6 +1907,37 @@ void main() {
     expect(selectionCount, 1);
     expect(tester.takeException(), isNull);
   });
+
+
+  // Test for #139871
+  // DropdownMenu throws exception when it is in any scrollable list view and scrolls quickly #139871
+  // https://github.com/flutter/flutter/issues/139871
```

Comment:
fly-by nit: convention is to just say:

```suggestion
  // Regression test for https://github.com/flutter/flutter/issues/139871.
```